packages/transport-webtransport/.aegir.js

```diff
@@ -1,14 +1,18 @@
+/* eslint-disable no-console */
 import { spawn, exec } from 'child_process'
-import { existsSync } from 'fs'
+import { existsSync } from 'node:fs'
+import os from 'node:os'
 import defer from 'p-defer'
 
 /** @type {import('aegir/types').PartialOptions} */
 export default {
   test: {
-    async before() {
+    async before () {
+      const main = os.platform() === 'win32' ? 'main.exe' : 'main'
+
       if (!existsSync('./go-libp2p-webtransport-server/main')) {
         await new Promise((resolve, reject) => {
-          exec('go build -o main main.go',
+          exec(`go build -o ${main} main.go`,
```

Comment:
I don't think these build files are being cleaned by `npm run clean`, which they probably should be

Reply:
TBH I want to remove this transport server entirely and just rely on precomplied binaries in the interop tests.